bodhi/server/models/models.py

```diff
@@ -1260,11 +1264,26 @@ def modify_bugs(self):
                     # https://github.com/fedora-infra/bodhi/issues/368#issuecomment-135155215
                     for bug in self.bugs:
                         if not bug.parent:
-                            log.debug("Closing tracker bug %d" % bug.bug_id)
-                            bug.close_bug(self)
+                            bad, good = self.get_bug_karma(bug)
+                            bug_karma = good + bad
```

Comment:
Suggestion: You could replace these two lines with a single but still readable line: ```bug_karma = sum(self.get_bug_karma(bug))``` since ```bad``` and ```good``` are just intermediate variables.